backend/lcfs/web/api/compliance_report/repo.py

```diff
@@ -158,37 +166,20 @@ async def check_compliance_report(
                 joinedload(ComplianceReport.organization),
                 joinedload(ComplianceReport.compliance_period),
                 joinedload(ComplianceReport.current_status),
+                joinedload(ComplianceReport.summary),
+                joinedload(ComplianceReport.fuel_supplies),
+                joinedload(ComplianceReport.other_uses),
+                joinedload(ComplianceReport.history).joinedload(
```

Comment:
Does this join history twice with its own different joins? Could we join all of them at once?

Reply:
The problem is that they're separate relationships so i don't know of a way to chain them from the one parent. I get errors like this when i tried it: "ORM mapped entity or attribute "ComplianceReportHistory.user_profile" does not link from relationship"

Let me know if there's a more optimized way to do this.